docroot/themes/custom/uids_base/scss/content/node--type--article.scss

```diff
@@ -149,6 +149,38 @@ article.node--type-article {
       margin-right: 0.3rem;
     }
   }
+
+  .block-field-blocknodearticlefield-image-caption {
+    font-style: italic;
+    z-index: 3;
+    position: relative;
+    padding: 1rem $mobile-width-gutter;
+    margin: -22.5px -22.5px $gutter -22.5px;
+
+    @include breakpoint(sm) {
+      color: #fff;
+      padding: 10px $gutter $gutter $gutter;
+      order: 3;
+      margin: 0 -22.5px -22.5px -22.5px;
+    }
+  }
+
+  .block-system-breadcrumb-block {
+    order: 2;
+  }
+
+  .block-field-blocknodearticletitle {
+    order: 3;
+    @include breakpoint(sm) {
+      order: 2;
+    }
+  }
```

Comment:
I think, but am not totally sure, that we can omit these, since their values will be inferred.

Reply:
Yup! Needed a specified `-1` added to the caption, but after that, the rest falls in line.